zavod/zavod/tests/fixtures/testdataset1/testdataset1.yml

```diff
@@ -2,6 +2,7 @@ entry_point: testentrypoint1
 title: OpenSanctions Validation Dataset
 prefix: osv
 hidden: true
+resolve: false
```

Comment:
This hits me as super surprising - that our main fixture is going to have one of the default features disabled. Does it not break any tests?

Reply:
yeah it definitely has a smell, and not a nice one.

Only two failed!

I need to go, but I'll fix when I'm back. This is not an urgent change